src/main/java/com/robobunny/SeekBarPreference.java

```diff
@@ -1,238 +0,0 @@
-
-
-public class SeekBarPreference extends Preference implements OnSeekBarChangeListener {
-
-    private final String TAG = getClass().getName();
-
-    private static final String ANDROIDNS="http://schemas.android.com/apk/res/android";
-    private static final String APPLICATIONNS="http://robobunny.com";
-    private static final int DEFAULT_VALUE = 50;
-    private final Logger log;
-
-    private int mMaxValue      = 100;
-    private int mMinValue      = 0;
-    private int mInterval      = 1;
-    private int mCurrentValue;
-    private String mUnitsLeft  = "";
-    private String mUnitsRight = "";
-    private SeekBar mSeekBar;
-
-    private TextView mStatusText;
-
-    public SeekBarPreference(@NonNull Context context, @NonNull AttributeSet attrs) {
-        super(context, attrs);
-        log = Logger.INSTANCE;
-        initPreference(context, attrs);
-    }
-
-    public SeekBarPreference(@NonNull Context context, @NonNull AttributeSet attrs, int defStyle) {
-        super(context, attrs, defStyle);
-        log = Logger.INSTANCE;
-        initPreference(context, attrs);
-    }
-
-    private void initPreference(Context context, @NonNull AttributeSet attrs) {
-        setValuesFromXml(attrs);
-        mSeekBar = new SeekBar(context, attrs);
-        mSeekBar.setMax(mMaxValue - mMinValue);
-        mSeekBar.setOnSeekBarChangeListener(this);
-
-        setWidgetLayoutResource(R.layout.seek_bar_preference);
-    }
-
-    private void setValuesFromXml(@NonNull AttributeSet attrs) {
-        mMaxValue = attrs.getAttributeIntValue(ANDROIDNS, "max", 100);
-        mMinValue = attrs.getAttributeIntValue(APPLICATIONNS, "min", 0);
-
-        mUnitsLeft = getAttributeStringValue(attrs, APPLICATIONNS, "unitsLeft", "");
-        String units = getAttributeStringValue(attrs, APPLICATIONNS, "units", "");
-        mUnitsRight = getAttributeStringValue(attrs, APPLICATIONNS, "unitsRight", units);
-
-        try {
-            String newInterval = attrs.getAttributeValue(APPLICATIONNS, "interval");
-            if(newInterval != null)
-                mInterval = Integer.parseInt(newInterval);
-        }
-        catch(Exception e) {
-            log.error(TAG, "Invalid interval value", e);
-        }
-
-    }
-
-    private String getAttributeStringValue(@NonNull AttributeSet attrs, String namespace, String name, String defaultValue) {
-        String value = attrs.getAttributeValue(namespace, name);
-        if(value == null)
-            value = defaultValue;
-
-        return value;
-    }
-
-    @Override
-    protected View onCreateView(ViewGroup parent) {
-        View view = super.onCreateView(parent);
-
-        // The basic preference layout puts the widget frame to the right of the title and summary,
-        // so we need to change it a bit - the seekbar should be under them.
-        LinearLayout layout = (LinearLayout) view;
-        layout.setOrientation(LinearLayout.VERTICAL);
-
-        return view;
-    }

```

Comment:
Third change: remove `onCreateView` override entirely; the default serves us fine now.